main.cpp

```diff
@@ -71,7 +71,20 @@ void *aligned_ptr(size_t base_alignment, size_t required_size) {
     }
     void *p = storage_ptr;
     size_t space = STORAGE_SIZE;
-    void *r = std::align(base_alignment, required_size, p, space);
+
+    /* std::align isn't available in GCC and clang until fairly
```

Comment:
Huh, that's really dumb. I though C++11 was implemented since forever ago in gcc, but I guess this only made it in a couple of years ago. Maybe something for psnip if it doesn't already have it?

Reply:
[GCC bug #57350](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=57350). It has been fixed for a long time, but Travis runs on Ubuntu 14.04, which is GCC 4.8.

There is probably something in Boost; it seems like most new C++ APIs get their start there. It's such trivial code, though, that I don't think it's worth pulling anything in.